test-apps/ui-test-app/src/frontend/index.tsx

```diff
@@ -212,7 +212,12 @@ export class SampleAppIModelApp {
 
     // initialize Presentation
     await Presentation.initialize({
-      activeLocale: IModelApp.i18n.languageList()[0],
+      presentation: {

```

Comment:
Would you mind checking `SampleAppIModelApp.testAppConfiguration.useLocalSettings` and if true use your `BrowserLocalStorage` option?

Reply:
done